src/Sarif.Multitool.Library/SuppressCommand.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.Readers;
+using Microsoft.CodeAnalysis.Sarif.Visitors;
+using Microsoft.CodeAnalysis.Sarif.Writers;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool
+{
+    public class SuppressCommand : CommandBase
+    {
+        public SuppressCommand(IFileSystem fileSystem = null) : base(fileSystem)
+        {
+        }
+
+        public int Run(SuppressOptions options)
+        {
+            try
+            {
+                Console.WriteLine($"Suppress '{options.InputFilePath}' => '{options.OutputFilePath}'...");
+                var w = Stopwatch.StartNew();
+
+                bool valid = ValidateOptions(options);
+                if (!valid)
+                {
+                    return FAILURE;
+                }
+
+                SarifLog currentSarifLog = PrereleaseCompatibilityTransformer.UpdateToCurrentVersion(FileSystem.FileReadAllText(options.InputFilePath),
+                                                                                                     options.Formatting,
+                                                                                                     out string _);
+
+                SarifLog reformattedLog = new SuppressVisitor(options.Justification,
+                                                              options.Alias,
+                                                              options.Guids,
+                                                              options.Timestamps,
+                                                              options.ExpiryInDays,
+                                                              options.Status).VisitSarifLog(currentSarifLog);
+
+                string actualOutputPath = CommandUtilities.GetTransformedOutputFileName(options);
+                if (options.SarifOutputVersion == SarifVersion.OneZeroZero)
+                {
+                    var visitor = new SarifCurrentToVersionOneVisitor();
+                    visitor.VisitSarifLog(reformattedLog);
+
+                    WriteSarifFile(FileSystem, visitor.SarifLogVersionOne, actualOutputPath, options.Formatting, SarifContractResolverVersionOne.Instance);
+                }
+                else
+                {
+                    WriteSarifFile(FileSystem, reformattedLog, actualOutputPath, options.Formatting);
+                }
+
+                w.Stop();
+                Console.WriteLine($"Supress completed in {w.Elapsed}.");
```

Comment:
like the idea to capture the execution time, not all commands implemented this, maybe we can move it to command base to work for all commands?